app/components/StatusCard/index.tsx

```diff
@@ -3,14 +3,14 @@ import { NumberOutput } from '@the-deep/deep-ui';
 
 import styles from './styles.css';
 
-export interface statusCardProps {
+export interface StatusCardProps {
```

Comment:
```suggestion
export interface Props {
```

If this is used outside, define the name during import.

Reply:
Done